apps/spruce/src/components/Banners/RepoTrackerBanner/RepoTrackerBanner.test.tsx

```diff
@@ -91,20 +91,20 @@ describe("repotracker banner", () => {
         <MockedProvider
           mocks={[projectWithError, adminUser, setLastRevision, projectNoError]}
         >
-          <RepotrackerBanner projectIdentifier="evergreen" />
+          <RepoTrackerBanner projectIdentifier="evergreen" />
         </MockedProvider>,
       );
       render(<Component />);
       await waitFor(() => {
-        expect(screen.queryByDataCy("repotracker-error-banner")).toBeVisible();
+        expect(screen.queryByDataCy("repoTracker-error-banner")).toBeVisible();
       });
-      expect(screen.queryByDataCy("repotracker-error-trigger")).toBeVisible();
+      expect(screen.queryByDataCy("repoTracker-error-trigger")).toBeVisible();
 
       // Open modal.
       // @ts-expect-error: FIXME. This comment was added by an automated script.
-      await user.click(screen.queryByDataCy("repotracker-error-trigger"));
+      await user.click(screen.queryByDataCy("repoTracker-error-trigger"));
```

Comment:
can you fix this error? all you have to do is change `queryByDataCy` => `getByDataCy`